reference-implementation/test/readable-byte-stream.js

```diff
@@ -1538,8 +1509,6 @@ test('ReadableByteStream: Multiple read(view), close() and respond()', t => {
     t.equals(view.byteLength, 0, '1st read: byteLength');
   });
 
-  t.equals(pullIntoCount, 1);
```

Comment:
I removed them because due to the addition of pull blocking by start method the expectations became stale.

I recovered some but realized we need to resolve https://github.com/whatwg/streams/issues/423 first.